app/models/game.rb

```diff
@@ -13,28 +13,28 @@ class Game < ApplicationRecord
 # Game rules of chess
 def initialize_board_white_pieces
   8.times do
-    Pawn.create(game_id: id, x_position: 2, y_position: 1, color: true)
+    Pawn.create(game_id: id, x_position: 1, y_position: 2, color: 'white')
   end
-  Rook.create(game_id: id, x_position: 1, y_position: 1, color: true)
-  Knight.create(game_id: id, x_position: 1, y_position: 2, color: true)
-  Bishop.create(game_id: id, x_position: 1, y_position: 3, color: true)
-  Queen.create(game_id: id, x_position: 1, y_position: 4, color: true)
-  King.create(game_id: id, x_position: 1, y_position: 5, color: true)
-  Bishop.create(game_id: id, x_position: 1, y_position: 6, color: true)
-  Knight.create(game_id: id, x_position: 1, y_position: 7, color: true)
-  Rook.create(game_id: id, x_position: 1, y_position: 8, color: true)
+  Rook.create(game_id: id, x_position: 1, y_position: 1, color: 'white')
+  Knight.create(game_id: id, x_position: 2, y_position: 1, color: 'white')
+  Bishop.create(game_id: id, x_position: 3, y_position: 1, color: 'white')
+  Queen.create(game_id: id, x_position: 4, y_position: 1, color: 'white')
+  King.create(game_id: id, x_position: 5, y_position: 1, color: 'white')
+  Bishop.create(game_id: id, x_position: 6, y_position: 1, color: 'white')
+  Knight.create(game_id: id, x_position: 7, y_position: 1, color: 'white')
+  Rook.create(game_id: id, x_position: 8, y_position: 1, color: 'white')
 end
 
 def initialize_board_black_pieces
   8.times do
-    Pawn.create(game_id: id, x_position: 7, y_position: 1, color: true)
+    Pawn.create(game_id: id, x_position: 7, y_position: 7, color: 'black')
   end
-  Rook.create(game_id: id, x_position: 8, y_position: 1, color: true)
-  Knight.create(game_id: id, x_position: 8, y_position: 2, color: true)
-  Bishop.create(game_id: id, x_position: 8, y_position: 3, color: true)
-  Queen.create(game_id: id, x_position: 8, y_position: 4, color: true)
-  King.create(game_id: id, x_position: 8, y_position: 5, color: true)
-  Bishop.create(game_id: id, x_position: 8, y_position: 6, color: true)
-  Knight.create(game_id: id, x_position: 8, y_position: 7, color: true)
-  Rook.create(game_id: id, x_position: 8, y_position: 8, color: true)
+  Rook.create(game_id: id, x_position: 8, y_position: 2, color: 'black')
```

Comment:
These should all be `y_position: 8` to put them in the top row of the board, above the black pawns you've created in row 7.